cmd/generator/main_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"testing"
+)
+
+type testCase struct {
+	name   string
+	regex  string
+	result string
+}
+
+func Test_generateCode(t *testing.T) {
```

Comment:
```suggestion
func TestGenerateCode(t *testing.T) {
```

[the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names](https://go.dev/doc/effective_go)

There is an exception for `Examples_`.

Reply:
Ok :)